github/ipld.yml

```diff
@@ -332,13 +332,6 @@ repositories:
     advanced_security: false
     allow_update_branch: false
     archived: false
-    collaborators:
```

Comment:
This one's interesting because it's for a private repository that was used for security release coordination with other teams. The people here don't contribute to the org but would have been added for visibility.

I _think_ it makes sense to have them pruned in a process like this. Ideally we'd just be using private forks off advisory drafts but sometimes it doesn't make sense, like for this one. The repo in question here is less than 60 months old and they're always going to be deleted regardless of how short you make it. If you run it too often then we're going to hit snags because they'd be added to a private repo and then removed as soon as the next run happens.

Reply:
Right now, one option to preserve access through and permanently opt out of the automated cleanup would be to add `KEEP` comments. It would look something like that:

```
      pull:
        # KEEP: Because ... 
        - alvin-reyes
        # KEEP: Because ... 
        - jennijuju
        # KEEP: Because ... 
        - TippyFlitsUK
      push:
        # KEEP: Because ... 
        - hsanjuan
```

Do you think it'd make sense to allow opting out entire repositories out of the automated cleanup?